_build_cfg.yml

```diff
@@ -192,6 +196,10 @@ Topics:
         File: persistent_storage_glusterfs
       - Name: Using OpenStack Cinder
         File: persistent_storage_cinder
+      - Name: Using AWS ElasticBlockStore
```

Comment:
Was this intentionally left camelcase? Looks like it's usually spaced out Elastic Block Store by Amazon.